### PR TITLE
ci: release on github and npmjs

### PR DESCRIPTION
**Pull request by vojtechsimetka**
For more info and how to use please see https://github.com/ethersphere/bee-js/pull/70

## TODO: 
- [x] test on personal fork of the repo (see [Github Action](https://github.com/vojtechsimetka/swarm-cli/runs/1816306410) and [npm package](https://www.npmjs.com/package/@vojtechsimetka/swarm-cli) )
- [x] verify npmjs token is set (@vandot ?)
- [x] verify github PAT is set (@vandot ?)

### CODE REVIEW

**Comment by vandot** [comment]
@vojtechsimetka added PAT as `REPO_GHA_PAT`

File: .github/workflows/release_github.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: GoogleCloudPlatform/release-please-action@v2
+        id: release
+        with:
+          token: ${{ secrets.REPO_GHA_PAT }}
+          release-type: node
+          package-name: bee-js
```
Comment (vandot) [inline .github/workflows/release_github.yaml:19]: @vojtechsimetka shouldn't be `swarm-cli`?

Comment (vojtechsimetka) [reply .github/workflows/release_github.yaml:19]: you are right! Thanks

Comment (vojtechsimetka) [reply .github/workflows/release_github.yaml:19]:
```suggestion
          package-name: swarm-cli
```

**Review by nugaon — APPROVED** [review]
LGTM!

**Pull request merged**